docs/application/dotnet/tutorials/certificates/creating-certificates.md

```diff
@@ -0,0 +1,116 @@
+# Creating Certificates
```

Comment:
Create Certificates

Reply:
done

Reply:
We didn't change writing style from the old style, starting with gerund, to new style, staring with verb, for all documents.
Before change all docs, please maintain consistency. Let's change the titles of all documents later.

Page title: # Creating Certificates

Reply:
I restored the modifications.